src/dos/cdrom.h

```diff
@@ -62,6 +62,8 @@
 #define AUDIO_DECODE_BUFFER_SIZE 16512u
 #define SAMPLES_PER_REDBOOK_FRAME (BYTES_PER_RAW_REDBOOK_FRAME / REDBOOK_BPS)
 
+constexpr auto dos_drives = 26;
```

Comment:
Constants are UpperCamelCase, and suggest the following change:

```cpp
// Number of letters in the English alphabet (A to Z)
constexpr auto MaxNumDosDriveLetters = 26;
```

Reply:
Thanks for changing, maybe add this too above it?

```
// Number of letters in the English alphabet (A to Z)
```

Reply:
Done. 👍